### ISSUE TITLE
Bug in npm scripts autocompletion using clink (can't handle backslash-escaped double quote)

### ISSUE THREAD

**Issue by ByteCommander** [cmderdev/cmder#1316]
I have lines like these in the `package.json` of one of my NodeJS projects, in the `scripts` section:

```
    "skillinfo-approve": "echo off & pushd skill-info && ( for /r %f in (*) do if not \"%~xf\" == \".md5\" md5sum.exe \"%f\" ) > .last-state.md5 & popd & echo Skill info marked as updated.",
    "lambda-view": "node -e \"console.log(JSON.stringify(JSON.parse(require('fs').readFileSync('./out/lambda-response.json','utf-8')),null,4))\" 2>nul || cat out/lambda-response.json",
```

Unfortunately these two lines break autocompletion, probably because they contain backslash-escaped double quotes:

```
λ npm run " [press TAB here]
 2>nul || cat out/lambda-response.json",^J    "lambda
 == \".md5\" md5sum.exe \"%f\" ) > .last-state.md5 & popd & echo Skill info marked as updated.",^J    "deploy
```

Can you please fix the LUA script responsible for the npm autocompletion and allow it to handle escaped double-quotes? Thank you a lot.

**Comment by ByteCommander** [comment]
Apparently it seems to be because Cmder uses an old version of https://github.com/vladimir-kotikov/clink-completions/blob/master/npm.lua - this should be updated.

**Comment by Stanzilla** [comment]
Usually @vladimir-kotikov does a PR to the project when he finished a new clink-completions release, so waiting for that.

**Comment by ByteCommander** [comment]
Just saying, there have been a few labeled releases already that include this fix, which are several months old. Maybe he forgot about it or whatever, but it could possibly be useful to become active on this end of the stream too... Anyway, thanks for replying.

**Comment by Stanzilla** [comment]
@ByteCommander we bundle 0.3.2 though, so maybe you are just on an old release?

**Comment by vladimir-kotikov** [comment]
@ByteCommander, I'm not quite sure I completely undersatnd so just to clarify - are you running into this problem w/ clink-completions@0.3.2? Could you make sure your npm.lua is identical to one from 0.3.2 release

**Comment by ByteCommander** [comment]
Hmm, apparently the bug I am talking about has been fixed with clink-completions release 3.1. It's an office setup that wasn't installed by me, so I guess I have to blame my sysadmins for deploying an outdated version. Sorry for the inconvenience, seems like I was just having an outdated version without noticing.
